shared/rakefiles/stackdriver.rb

```diff
@@ -68,6 +68,36 @@ def get_alert_policy_identifier(alert_policy)
   return result
 end
 
+def compare_alert_policies(stackdriver_alert_policy, alert_policy)
+  stackdriver_alert_policy = JSON.parse(stackdriver_alert_policy.to_hash.to_json)
+  ["name", "creation_record", "mutated_by", "mutation_record"]. each do |attribute|
+    stackdriver_alert_policy.delete(attribute)
+  end
+  stackdriver_alert_policy.delete("documentation") if stackdriver_alert_policy["documentation"] == nil
+  stackdriver_alert_policy["conditions"].each do |condition|
```

Comment:
Well, It is not a json, it is `Google::Monitoring::V3::AlertPolicy` object – we collapse it into json for the matter of convenience. 
And `conditions` is a [mandatory attribute](http://googleapis.github.io/google-cloud-ruby/docs/google-cloud-monitoring/v0.29.0/Google/Monitoring/V3/AlertPolicy.html) for any alert policy. I have very high doubts that it is going to change, and in case it changes – all our alert policy json configs will become invalid too.